client/tooltips.js

```diff
@@ -0,0 +1,156 @@
+
+    mode: "Concentration - Species will only be represented as deterministic.<br>Population - Species will only be represented as stochastic.<br>Hybrid "+
+        "Concentration/Population - Allows a species to be represented deterministically and/or stochastically.",
+
+    switchValue: "Switching Tolerance - Tolerance level for considering a dynamic species deterministically, value is compared to an estimated sd/mean population of a "+
+        "species after a given time step. This value will be used if a switch_min is not provided.<br>Minimum Value For Switching - Minimum population value at which "+
+        "species will be represented as Concentration."
+  },
+  parametersEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+    
+    expression: "A parameter value or a mathematical expression calculating the parameter value.",
+    
+    annotation: "An optional note about a parameter.",
+    
+    remove: "A parameter may only be removed if it is not used in any reaction, event assignment, or rule."
+  },
+  reactionsEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+    
+    annotation: "An optional note about the reaction.",
+    
+    rate: "The rate of the mass-action reaction.",
+    
+    propensity: "The custom propensity expression for the reaction.",
+    
+    reactant: "The reactants that are consumed in the reaction, with stoichiometry.",
+    
+    product: "The species that are created by the reaction event, with stoichiometry.",
+    
+    reaction: "For a species that is NOT consumed in the reaction but is part of a massaction reaction, add it as both a reactant and a product.\nMass-action reactions "+
+        "must also have a rate term added. Note that the inputrate represents the mass-action constant rate independent of volume."
+  },
+  eventsEditor: {
+    name: "Names for species, parameters, reactions, events, and rules must be unique.",
+    
+    annotation: "An optional note about an event.",
+    
+    triggerExpression: "The trigger expression can be any mathematical expression which evaluates to a boolean value in a python environment (i.e. t==50).  This "+
+        "expression is evaluable within the model namespace, and any variable (Species, Parameters, etc.) can be referenced in the expression.  Time is represented "+
+        "with the lower case variable 't'. An event will begin execution of assignments (or delay, if any) once this expression changes from 'False' to 'True.'",
+    
+    delay: "contains math expression evaluable within model namespace. This expression designates a delay between the trigger of an event and the execution of its assignments.",
+    
+    priority: "Contains a math expression evaluable within model namespace.  This expression designates execution order for events which are executed simultaneously.",
```

Comment:
"Contains a math expression evaluable within model namespace." is this true @seanebum?

Reply:
Change to: "Typically a number: 1,2,etc. that designates execution order for events which are executed simultaneously. Can be a math expression evaluable within model namespace"